react-native-git-upgrade/cli.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+require('babel-register')({
+  presets: [
+    require('babel-preset-es2015-node'),
+    require('babel-preset-stage-3')
```

Comment:
No, it doesn't. Babel is set in motion only if you required the [cli.js](https://github.com/ncuillery/react-native/blob/master/local-cli/cli.js) file.

If you do that it would probably work (it worked when I tried direct access to the upgrade command). 

Note that `node local-cli/someScript.js` will start immediately. It's `babel-register` which causes the ~2-3 seconds delay when starting a `react-native whatever` command.
It makes me raising questions I intend to ask later like:
- Couldn't we get rid of babel-register and publish transpiled modules ?